anaAllChambersThresh.py

```diff
@@ -0,0 +1,149 @@
+#!/bin/env python
+
+def launch(args):
+  return launchArgs(*args)
+
+def launchArgs(link,scandate_scurve,scandate_thresh,cName,cType):
+    import datetime,os,sys
+    import subprocess
+    from subprocess import CalledProcessError
+    from chamberInfo import chamber_config
+    from gempython.utils.wrappers import runCommand
+
+    #print "launchArgs(%s,%s,%s,%s,%s)"%(link,scandate_scurve,scandate_thresh,cName,cType)
+
+    dataPath = os.getenv('DATA_PATH')
+    elogPath = "%s/%s"%(os.getenv('ELOG_PATH'),scandate_thresh)
+   
+    #filename_scurve = "%s/%s/scurve/%s/SCurveData/SCurveFitData.root"%(dataPath,cName,scandate_scurve)
+    filename_scurve = "%s/%s/trim/z4.000000/%s/SCurveData_Trimmed/SCurveFitData.root"%(dataPath,cName,scandate_scurve)
```

Comment:
If we make "z4.000000" a configurable parameter this should be sufficient yes?  The remainder of the structure seems to be required to fit the file structure setup on the gemdaq machine:

e.g.
filename_scurve = "%s/%s/trim/z4.000000/%s/SCurveData_Trimmed/SCurveFitData.root"%(dataPath,cName,scandate_scurve)

To:
+    filename_scurve = "%s/%s/trim/%s/%s/SCurveData_Trimmed/SCurveFitData.root"%(dataPath,cName,trimRange,scandate_scurve)

The "SCurveData_Trimmed/SCurveFitData.root" should remain fixed as it is in anaAllChambers.py line 15:

https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/master/anaAllChambers.py